h2o-genmodel/build.gradle

```diff
@@ -24,6 +24,8 @@ javadoc {
 dependencies {
   testCompile "com.google.guava:guava:16.0.1"
   testCompile 'org.mockito:mockito-core:2.18.3'
+  testCompile 'org.powermock:powermock-module-junit4-legacy:1.7.1' 
```

Comment:
did not see this used

Reply:
@honzasterba Because it is quite difficult to use mocks in our project. But yes, I was planning to remove this of course.

Reply:
yes, mocking in current state of the codebase is basically impossible